src/workloads/transactions/templates/LLTMixedLowLtc.tmpl

```diff
@@ -124,7 +124,7 @@ GlobalDefaults:
   PtcRemoveOperation: &PtcRemoveOperation
     OperationName: deleteOne
     OperationCommand:
-      Filter: {price: {'$gte' : {^RandomDouble: { min: 0.0, max: 500.0 }}}}
+      Filter: {'price': {'$gte' : {^RandomDouble: { min: 0.0, max: 1000.0 }}}}
```

Comment:
Are quotes actually needed or desired here? I think I saw Brad taking them out of his PR at request from Ryan, but could be wrong.

Reply:
Oh, it's because it's templated, isn't it?

Reply:
no it was more to make them consistent (but it looks like I picked the wrong thing to standardize on).

I will leave it for the moment though.